.github/workflows/pr_helper.yml

```diff
@@ -1,8 +1,7 @@
 name: PR helper
 on:
-  pull_request:
-    types:
-      - opened
+  pull_request_target:
+    types: [opened, reopened]
```

Comment:
Temporarily add trigger on reopened. It will be removed after PR helper is stable.

Reply:
Interesting syntax!
I did not realize `YAML` allows lists like this and always thought it requires
```
- opened
- reopened
```
Good to learn!

Reply:
Adding sample document: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
![image](https://github.com/google/oss-fuzz/assets/39108850/c37357e6-4bfe-4d40-a757-31b572cf34db)